### PR TITLE
Fix add Task Error

### LINKED ISSUE #162: Task with the same name is considered the name when other fields are different

### PR DESCRIPTION
**Pull request by hengyongming**
1. add n/Finish 2103T Tutorial desc/Draw UML diagram date/2021-04-13 10:30 t/CS2103T
2.  add n/Finish 2103T Tutorial desc/Draw UML diagram date/2021-04-13 15:30 t/CS2103T

Although the 2 task has the same name, the other fields are different. And thus it should not be a duplicated task.

However, if a task has the same attributes except for status, it is considered a duplicate task.

### CODE REVIEW

File: src/test/data/JsonSerializableTaskifyParserTest/duplicateTaskTaskify.json [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
-        "friends"
+        "Lab"
       ]
     },
     {
-      "name": "Alice Pauline",
-      "description": "2107 ctf",
+      "name": "CS2106 Lab",
+      "description": "Lab on Synchronization",
       "address": "4th street",
```
Comment (CSmortal) [inline src/test/data/JsonSerializableTaskifyParserTest/duplicateTaskTaskify.json]: line 15 should be deleted

**Review by CSmortal — CHANGES_REQUESTED** [review]

File: src/test/data/JsonSerializableTaskifyParserTest/duplicateTaskTaskify.json (hunk 1 above)
Comment (hengyongming) [reply src/test/data/JsonSerializableTaskifyParserTest/duplicateTaskTaskify.json]: alright will fix that thanks

**Review by khiaxeng — APPROVED** [review]
LGTM

**Review by CSmortal — APPROVED** [review]
LGTM

**Pull request merged**